packages/core/admin/admin/src/components/DescriptionComponentRenderer.tsx

```diff
@@ -134,5 +144,20 @@ function getCompId<T, K>(comp: DescriptionComponent<T, K>): string {
   return id;
 }
 
+const useShallowCompareMemoize = <T,>(value: T): Array<T | undefined> => {
```

Comment:
The coma after T seems forgotten

Reply:
This is the same lint error you've asked about before, in `tsx` files it assumes its JSX which it's not :(